coresimd/x86/cpuid.rs

```diff
@@ -44,8 +49,11 @@ pub struct CpuidResult {
 /// [wiki_cpuid]: https://en.wikipedia.org/wiki/CPUID
 /// [intel64_ref]: http://www.intel.de/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-instruction-set-reference-manual-325383.pdf
 /// [amd64_ref]: http://support.amd.com/TechDocs/24594.pdf
+///
+/// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=__cpuid_count)
```

Comment:
This is not an "official" Intel intrinsic, so this link returns empty. IIRC at least the cpuid intrinsics and the read/writeeflags intrinsics will suffer from this. There should be a list of these or their modules in the `stdsimd-verify` module.